src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/magical/talismans/MagicianTalisman.java

```diff
@@ -31,23 +31,24 @@
 public class MagicianTalisman extends Talisman {
 
     private final ItemSetting<Boolean> allowEnchantmentBooks = new ItemSetting<>("allow-enchantment-books", false);
-    private final ItemSetting<Boolean> allowCustomEnchantments = new ItemSetting<>("allow-custom-enchants", false);
 
     private final Set<TalismanEnchantment> enchantments = new HashSet<>();
 
     @ParametersAreNonnullByDefault
     public MagicianTalisman(SlimefunItemStack item, ItemStack[] recipe) {
         super(item, recipe, false, false, "magician", 80);
 
-        addItemSetting(allowEnchantmentBooks, allowCustomEnchantments);
+        addItemSetting(allowEnchantmentBooks);
 
         for (Enchantment enchantment : Enchantment.values()) {
             /* 
              * Stop custom enchant plugins appearing
              * We also want to stop some stupid plugins which register as Minecraft enchants
              * but also go above the max enchant level.
              */
-            if ((allowCustomEnchantments.getValue() || enchantment.getKey().getNamespace().equals(NamespacedKey.MINECRAFT))) {
+            if ((SlimefunPlugin.getCfg().getBoolean("allow-custom-enchants")
```

Comment:
Should probably cache it, like all other config settings.
`SlimefunRegistry` is currently used for this